airflow/contrib/operators/sqoop_operator.py

```diff
@@ -59,12 +66,18 @@ def __init__(self,
                  properties=None,
                  hcatalog_database=None,
                  hcatalog_table=None,
+                 create_hcatalog_table=False,
```

Comment:
adding option to create the hcatalog table

Reply:
👍